### DISCUSSION TITLE
p.203 リスト19.1のコンパイルエラー

### DISCUSSION THREAD

**Discussion by kdnakt** [budougumi0617/go_todo_app#36] [Ideas]
```diff
+ err := at.Repo.AddTask(ctx, at.DB, t)
```

とありますが、リスト17.7の実装では少し上の行で既にerrを宣言しているためか、コンパイルエラーとなりました

<img width="504" alt="image" src="https://user-images.githubusercontent.com/5971361/184758898-2e02cb44-b67f-46a0-82e2-09e02bd5b483.png">

変更箇所を以下のようにするか、リスト17.7のバリデーションのコードを[リポジトリのサンプルコード](https://github.com/budougumi0617/go_todo_app/blob/v1.0.1/_chapter19/section75/handler/add_task.go#L30)に合わせて1行で書く必要がありそうです。

```diff
+ err = at.Repo.AddTask(ctx, at.DB, t)
```

**Comment by budougumi0617** [comment]
ありがとうございます！
普段でも `if` 文と分けて書かないので横幅気にして紙面上だけで分けてしまったのかもしれないです。追って正誤表更新します！

**Comment by budougumi0617** [comment]
ご指摘ありがとうございました。正誤表に反映しました。
- https://github.com/budougumi0617/go_todo_app/pull/46

// 今いただいている他の未対応の指摘もひととおり対応しだいリリースタグつけようと思います 🙇

**Comment by budougumi0617** [comment]
正誤表を更新しリリースタグをつけました。ご報告ありがとうございました。
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.5
https://github.com/budougumi0617/go_todo_app/blob/v1.0.5/errata.md
